commerce_coordinator/apps/rollout/constants.py

```diff
@@ -0,0 +1,9 @@
+# .. toggle_name: redirect_to_commercetools_checkout
+# .. toggle_type: waffle_flag
+# .. toggle_default: False
+# .. toggle_description: Determines if user will be redirected to commerce tools or legacy
+# .. toggle_creation_date: 2023-11-15
+# .. toggle_tickets: SONIC-100
+
+# TODO: Create flag in stage/prod
+REDIRECT_TO_COMMERCETOOLS_CHECKOUT = 'redirect_to_commercetools_checkout'
```

Comment:
Can we please move waffle flags to a separate file instead of constants? Ref: https://github.com/edx/subscriptions/blob/main/subscriptions/apps/subscriptions/waffle.py

Reply:
lets add it in `app/rollout`